source/lib-intro.tex

```diff
@@ -1324,10 +1324,8 @@
 
 \rSec2[using]{Using the library}
 
-\rSec3[using.overview]{Overview}
-
 \pnum
-This section describes how a \Cpp program gains access to the facilities of the
+This subclause describes how a \Cpp program gains access to the facilities of the
```

Comment:
How about: keep the `Overview` subclause, and use "Subclause \ref{using} describes" here.

Reply:
Fixed.